resources/assets/coffee/status-map.coffee

```diff
@@ -0,0 +1,121 @@
+    force = d3.layout.force()
+
+    @maxHeight = 597
+    @fontSize = 17
+
+    @colors =
+      up: '#acd300'
+      down: '#ff0000'
+
+    @svg = d3.select(element)
+            .append('svg')
+            .attr("width", '100%')
+            .attr("height", '100%')
+            .attr('viewBox','0 0 ' + width + ' ' + (@height - @offset))
+            .attr('preserveAspectRatio','xMinYMin')
+
+    @update(servers)
+
+    # resize map's view
+    resize = () ->
+      width = $(element).width()
+      height = (width*9.3/21)
+      $(element).height(height)
+      force.size([width, height]).resume()
+      return
+
+    d3.select(window).on('resize', resize)
+
+  update: (servers) =>
+    g = @svg.append('g')
+           .attr("transform", "translate(0, 0)")
+
+    img = g.append('svg:image')
+           .attr('xlink:href', '../images/backgrounds/world_map.svg')
+           .attr('width', '100%')
+           .attr('height', '100%')
+           .attr('x', 0)
+           .attr('y', @scale(@offset / 2))
+
+    # inserts each circle + name and player number
+    for server in servers
+      @svg.append('ellipse')
+         .attr('rx', @scale(8))
+         .attr('ry', @scale(8))
+         .attr('cx', @scale(server.x))
+         .attr('cy', @scale(server.y))
+         .style('fill', 'none')
+         .attr('stroke-miterlimit', 0)
+         .attr('stroke-width', @scale(5))
+         .attr('stroke', @colors[server.state])
+
+      # the "O" that grows
+      grower = @svg.append('ellipse')
+                  .attr('rx', @scale(8))
+                  .attr('ry', @scale(8))
+                  .attr('cx', @scale(server.x))
+                  .attr('cy', @scale(server.y))
+                  .style('fill', 'none')
+                  .attr('stroke-miterlimit', 0)
+                  .attr('stroke-width', @scale(5))
+                  .attr('stroke', @colors[server.state])
+
+      ghg = @scale(28)
+      gho = @scale(8)
+      grower.on 'mouseover', ->
+        d3.select(this)
+          .transition()
+          .duration(1000)
+          .attr('rx', ghg)
+          .attr('ry', ghg)
+          .style('opacity', 0).each 'end', (->
+            d3.select(this)
+              .transition()
+              .duration(1)
+              .attr('rx', gho)
+              .attr('ry', gho)
+              .style('opacity', 1)
+          )
+
+      @svg.append('text')
+         .text(server.name)
+         .attr('x', @scale(server.x))
+         .attr('y', @scale(server.y - 40))
+         .attr('fill', 'white')
+         .attr('text-anchor', 'middle')
+         .style('font-size', @scale(@fontSize))
+
+      @svg.append('text')
+         .text("#{server.players} players #{(if server.state == 'down' then 'offline' else 'online')}")
+         .attr('x', @scale(server.x))
+         .attr('y', @scale(server.y - 20))
+         .attr('fill', 'white')
+         .attr('text-anchor', 'middle')
+         .style('font-size', @scale(@fontSize))
+
+  scale: (val) =>

```

Comment:
then you load the page at a lower resolution and everything is at the wrong place